common/scala/src/main/resources/kafka.conf

```diff
@@ -0,0 +1,24 @@
+kafka {
```

Comment:
Why do we create a new kafka.conf? We could simply add a whisk.kafka section the the application.conf

Reply:
It struck me as more modular to have a conf files for the various sub-components and include them in application.conf (similar to how logging is already done).  I don't have strong opinions on this, but we should decide what we want to do and document it.

@markusthoemmes, is there some write up of conventions for how we should be using pureconfig?  One of my todos for the next couple of weeks is to migrate as much as possible from ansible to pureconfig to reduce the number of build breaks in kube.  We had 4 separate kube build breaks last week due to variables been added/updated in ansible configs and not replicated to the kube project.

Reply:
@dgrove-oss yep, there is: https://github.com/apache/incubator-openwhisk/blob/master/docs/dev/configuration.md

We should maybe add that all openwhisk related configuration should go under "whisk" to prevent potential clashes with other modules (reactive kafka in this case).

Reply:
ok.  So all the overrides still happen via environment variables (in ansible or yaml files for kube, etc).  So there is less value for a separate .conf file for different subsystems.  Should I merge kafka.conf into application.conf as suggested by @cbickel then?